src/modules/manage.js

```diff
@@ -0,0 +1,91 @@
+import dotsIcon from '../icons/dots.png';
+
+const icon3 = new Image();
+icon3.src = dotsIcon;
+const toDos = [];
+
+export default class ToDo {
+  constructor(todo) {
+    this.description = todo;
+    this.index = toDos.length + 1;
+    this.id = Date.now().toString();
+    this.completed = false;
+  }
+
+  static add() {
+    const alert = document.querySelector('.alert');
+    const addIcon = document.getElementById('add-icon');
+    const doInput = document.getElementById('do-input');
+    alert.style.display = 'none';
+    addIcon.addEventListener('click', () => {
+      let toDos = [];
+      toDos = JSON.parse(localStorage.getItem('toDos') || '[]');
+      if (doInput.value !== '') {
+        const newTask = new ToDo(doInput.value);
+        toDos.push(newTask);
+        toDos.forEach((obj, i) => {
+          obj.index = i + 1;
+        });
+        localStorage.setItem('toDos', JSON.stringify(toDos));
+        doInput.value = '';
+        window.location.reload();
+      } else {
+        alert.innerText = 'You have to do something!';
+        alert.style.display = 'flex';
+      }
+    });
+  }
+
+  static show() {
+    const todoList = document.getElementById('do-list');
+    const toDos = JSON.parse(localStorage.getItem('toDos') || '[]');
+    if (toDos.length > 0) {
+      toDos.forEach((task) => {
+        todoList.innerHTML += `
+        <li>
+          <div class="item-info">
+            <input type="checkbox">
+            <input id="${task.index}" class="task-item" value="${task.description}">
+          </div>
+          <span id="item-icon">
+            <img src="${icon3.src}" class="remove" id="${task.id}">
```

Comment:
- **[OPTIONAL]** Instead of the three dots I think it would be better to use a more suitable icon that indicates removal or deletion, like a trash can or x-mark